google/cloud/aiplatform/experimental/vertex_model/utils/source_utils.py

```diff
@@ -43,7 +43,14 @@ def _make_class_source(obj: Any):
     return "\n".join(source_maker.source)
 
 
-def _make_source(cls_source: str, cls_name: str, instance_method: str):
+def _make_source(
+    cls_source: str,
+    cls_name: str,
+    instance_method: str,
+    pass_through_params,
```

Comment:
Type hints on `pass_through_params`, `param_name_to_serialized_info`, and `obj`.